contract.md

```diff
@@ -4,6 +4,11 @@
 
 _At a high level, what do you most want to get out of this project, and why?_
 
+I would hope to get an understanding of the different ways Domain Specific Languages are made and what techniques can be used to make them. While we’ve been reading about Domain Specific Language design and have even made a DSL in the form of our Piconot language, I do think getting to explore different design principles and getting to run into pitfalls of DSL design naturally will help me learn about DSL design, which I can then use in later projects. I’d also love to put into practice what we’ve been learning about in DSL’s, like investigating the benefits and drawbacks of internal/external DSL’s, ease of use, natural language influence, and which programming languages are best suited to particular needs.
+	I also hope to have a project at the end that I will be proud of. Naturally I’m very interested in this topic (calendar/schedule making) as it always seems needlessly complicated to create a schedule in Google Calendar. I would want something that not only would I be proud to have made and show off, but also one I would want to use to fix that problem of needlessly complicated schedule creation. To elaborate on this, it feels like I need to click through too many options in Google Calendar to make a new event and time/dates may change when I go to edit them so I need to edit the time two or three times for any given event. I hope that by making a DSL that would easily allow for the creation of events by writing to a text file I can have something that would be useful not only to myself but to others as well.
+	Something else I’d want to get out of the project is a frontend user interface. While I’ve already discussed this with professor Weidermann, and it is at best a stretch goal if time allows for it, I would hope to be able to create a preview window similar to how context free implements a preview window. I do have some experience with creating desktop apps using Javascript frameworks, so while I would love to do something like that, I do acknowledge that it’s not required and will only be investigated if time allows for it.
```

Comment:
I think a frontend is also a stretch goal - I'm facing a similar position myself - where there's a set of tasks that fulfill the requirements and scope and I would be proud of it. But also that having a frontend would be very cool :)